joblib/parallel.py

```diff
@@ -1048,6 +1048,7 @@ def _batched_calls_reducer_callback():
         finally:
             if hasattr(self._backend, 'stop_call'):
                 self._backend.stop_call()
+                time.sleep(0.01)
```

Comment:
Actually we do not want to do this in general: we want to be `Parallel` calls to be as fast as possible on fast backends (e.g. the threading backend). So we need to fix the issue specifically for the dask backend.

Reply:
I was about to push a commit for this :)